service/src/main/java/uk/gov/hmcts/reform/fpl/service/translations/TranslationRequestService.java

```diff
@@ -47,13 +49,17 @@ public void sendRequest(CaseData caseData, Optional<LanguageTranslationRequireme
         byte[] originalDocumentContent = documentDownloadService.downloadDocument(document.getBinaryUrl());
 
         EmailAttachment originalDocument = document(
-            URLConnection.guessContentTypeFromName(document.getFilename()),
-            originalDocumentContent,
+            defaultIfNull(URLConnection.guessContentTypeFromName(document.getFilename()), "application/octet-stream"),
+            Arrays.copyOf(originalDocumentContent, originalDocumentContent.length),
```

Comment:
why do we need to create a copy of the array? is it altered in some way?

Reply:
👍